gusto/configuration.py

```diff
@@ -85,9 +87,21 @@ class ShallowWaterParameters(Configuration):
 class EadyParameters(Configuration):
 
     """
-    Physical parameters for nonlinear eady
+    Physical parameters for Incompressible Eady
     """
     Nsq = 2.5e-05  # squared Brunt-Vaisala frequency (1/s)
     dbdy = -1.0e-07
     H = None
-    geopotential = False  # use geopotential for gravity term
+    f = None
+
+
+class CompressibleEadyParameters(CompressibleParameters, EadyParameters):
+
+    """
+    Physical parameters for Compressible Eady
+    """
+    g = 10.
```

Comment:
This is to compare the results with the previous Eady papers that use this value, although this would not affect the results much.